camel-sap/pom.xml

```diff
@@ -18,73 +24,367 @@
+		<maven-dependency-plugin-version>3.1.1</maven-dependency-plugin-version>
+		<maven-surefire-plugin-version>2.22.1</maven-surefire-plugin-version>
+		<maven-resources-plugin-version>3.1.0</maven-resources-plugin-version>
+
 		<camel-version>2.17.0.redhat-630133</camel-version>
 		<camel.sap.plugin.version>6.3.0-SNAPSHOT</camel.sap.plugin.version>
 
-		
-		<!-- OSGi bundles properties -->
-		<fuse.osgi.bundle.name>${project.name}</fuse.osgi.bundle.name>
-		<fuse.osgi.import.fabric.version>
-		    version="[$(version;==;${fuse.osgi.version}),$(version;=+;${fuse.osgi.version}))"
-		</fuse.osgi.import.fabric.version>
-		<fuse.osgi.import.strict.version>
-		    version="[$(version;===;${fuse.osgi.version}),$(version;==+;${fuse.osgi.version}))"
-		</fuse.osgi.import.strict.version>
-		<fuse.osgi.import.default.version>[$(version;==;$(@)),$(version;+;$(@)))</fuse.osgi.import.default.version>
-		<fuse.osgi.import.defaults>
-		    org.springframework.osgi.*;version="[1.2,2)",
-		    org.springframework.*;version="[3.1,4)",
-		    org.apache.commons.logging.*;version="[1.1,2)",
-		    org.apache.camel.*;version="[2.13,3)",
-		    org.apache.felix.gogo.commands.*;version="[0.10,1)",
-		    org.apache.felix.service.command.*;version="[0.10,1)",
-		    org.osgi.service.component.*;version="[1.1,2.0)",
-		    scala*;version="[2.11,2.12)"
-		</fuse.osgi.import.defaults>
-		<fuse.osgi.import.before.defaults/>
-		<fuse.osgi.import.additional/>
-		<fuse.osgi.import.pkg>
-		    io.fabric8.*;${fuse.osgi.import.fabric.version},
-		    ${fuse.osgi.import.before.defaults},
-		    ${fuse.osgi.import.defaults},
-		    ${fuse.osgi.import.additional},
-		    *
-		</fuse.osgi.import.pkg>
-		<fuse.osgi.activator/>
-		<fuse.osgi.failok>false</fuse.osgi.failok>
-		<fuse.osgi.private.pkg></fuse.osgi.private.pkg>
-		<fuse.osgi.export>io.fabric8.*;version=${fuse.osgi.version};-noimport:=true</fuse.osgi.export>
-		<fuse.osgi.split.pkg>-split-package:=first</fuse.osgi.split.pkg>
-		<fuse.osgi.import>${fuse.osgi.import.pkg}</fuse.osgi.import>
-		<fuse.osgi.dynamic/>
-		<fuse.osgi.symbolic.name>${project.groupId}.${project.artifactId}</fuse.osgi.symbolic.name>
-		<fuse.osgi.exclude.dependencies>false</fuse.osgi.exclude.dependencies>
-		<fuse.osgi.resource>{maven-resources}</fuse.osgi.resource>
-		<fuse.osgi.embed.dependency>!*</fuse.osgi.embed.dependency>
-		<fuse.osgi.embed.transitive>false</fuse.osgi.embed.transitive>
-		<fuse.osgi.require.bundle />
-		<fuse.osgi.capabilities.provide />
-		<fuse.osgi.capabilities.require />
-			
+		<sap.idoc.lib.version>3.0.10</sap.idoc.lib.version>
+		<sap.jco.lib.version>3.0.11</sap.jco.lib.version>
+
+		<sap.redhat.version.postfix>redhat_2</sap.redhat.version.postfix>
+
+		<sap.idoc.wrapper.version>${sap.idoc.lib.version}.${sap.redhat.version.postfix}</sap.idoc.wrapper.version>
+		<sap.jco.wrapper.version>${sap.jco.lib.version}.${sap.redhat.version.postfix}</sap.jco.wrapper.version>
+
+		<jaxb.core.version>2.3.0.1</jaxb.core.version>
```

Comment:
on 6.3 stream it was 2.2.11, I think it is something that needs to stay aligned with other Fuse dependencies

https://github.com/jboss-fuse/fuse-components/blob/94a5d126f71d7b9158210dd7c2248a345e4d428e/camel-sap/pom.xml#L181